### PR TITLE
Network broker: log verbosity howto and gentle warning

### PR DESCRIPTION
**Pull request by armchairlinguist**
Add note on how to adjust logging on the broker, and warn that it can lead to high memory usage if the req/res are large.

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR

### CODE REVIEW

**Review by vivekkhimani — APPROVED** [review]

File: docs/semgrep-ci/network-broker.md [hunk 1]
```diff
@@ -125,6 +125,34 @@ You can check the logs for Semgrep Network Broker by running:
 
 <pre class="language-console"><code>sudo docker logs <span className="placeholder">CONTAINER_ID</span></code></pre>
 
+### Adjusting log verbosity
+
+The Semgrep Network broker can log details of the proxied requests and responses for troubleshooting. Normally, these should only be collected when working to identify an issue. If requests and responses are large, they can occupy significant memory in the tunnel.
+
+In your broker configuration, that looks like:
```
Comment (s-santillan) [inline docs/semgrep-ci/network-broker.md:132]: Is this an example we want people to copy?

**Review by s-santillan — APPROVED** [review]

File: docs/semgrep-ci/network-broker.md (hunk 1 above)
Comment (armchairlinguist) [reply docs/semgrep-ci/network-broker.md:132]: yep, they can safely copy this example. the one below is also syntactically fine although wouldn't be very useful

Comment (s-santillan) [reply docs/semgrep-ci/network-broker.md:132]: Got it. It sounded a little hypothetical (or rather I wasn't sure if I would use it) - maybe we can be more explicit that the reader can use the snippet? Minor suggestion only.

Comment (armchairlinguist) [reply docs/semgrep-ci/network-broker.md:132]: How would you phrase that? I don't have great idea of how to make it more clear, especially given it follows a caution only to use it for debugging. :)

Comment (s-santillan) [inline docs/semgrep-ci/network-broker.md:132]: Maybe:

> To log additional details, add this snippet to your broker configuration:

Comment (armchairlinguist) [reply docs/semgrep-ci/network-broker.md:132]: Valid, but I don't know how well it works with the warning above 🤔 
I'd prefer to merge it as is, but I don't feel strongly - if you prefer this, I'm happy to accept it as a suggestion / push and then merge.

Comment (s-santillan) [reply docs/semgrep-ci/network-broker.md:132]: Yeap, The specific bit I was thinking about was "that looks like" - it seemed ambiguous. There's no indication of "if i want to troubleshoot, I should copy this thing" - if you are troubleshooting something, additional logging is a valid step to take even if it doesn't always lead to the solution.

I would have suggested "To log additional details for troubleshooting, add this snippet..." However, the text says "normally" used for troubleshooting - so there are other cases we may not know why a user wants to add logs. I will make the suggestion, thanks for being open to it.

Comment (s-santillan) [reply docs/semgrep-ci/network-broker.md:132]:
```suggestion
To log additional details, add this snippet to your broker configuration:
```

**Pull request merged**